src/Data/Fin/Subset/Properties.agda

```diff
@@ -203,6 +256,171 @@ p⊆q⇒∣p∣<∣q∣ {p = outside ∷ p} {inside  ∷ q} p⊆q = ≤-step (p
 p⊆q⇒∣p∣<∣q∣ {p = inside  ∷ p} {outside ∷ q} p⊆q = contradiction (p⊆q here) λ()
 p⊆q⇒∣p∣<∣q∣ {p = inside  ∷ p} {inside  ∷ q} p⊆q = s≤s (p⊆q⇒∣p∣<∣q∣ (drop-∷-⊆ p⊆q))
 
+
+------------------------------------------------------------------------
+-- _⊂_
+
+p⊂q⇒p⊆q : ∀ {n} → {p q : Subset n} → p ⊂ q → p ⊆ q
+p⊂q⇒p⊆q = proj₁
+
+⊂-trans : ∀ {n} → Transitive (_⊂_ {n})
+⊂-trans (p⊆q , x , x∈q , x∉p) (q⊆r , _ , _ , _) = ⊆-trans p⊆q q⊆r , x , q⊆r x∈q , x∉p
+
+⊂-trans-⊆ : ∀ {n} → Trans (_⊂_ {n}) (_⊆_ {n}) (_⊂_ {n})
```

Comment:
I think the standard name for this is `⊂-⊆-trans` (e.g. see [Data.Rational](https://github.com/agda/agda-stdlib/blob/b1cdd5d7dddf5251d5eda0cbf3711fa2bbd3bbb5/src/Data/Rational/Properties.agda#L357)). I think(?) you could also make the type clearer as `Trans {A = Subset n} _⊂_ _⊆_ _⊂_`

Reply:
Done.